jdaviz/configs/mosviz/plugins/viewers.py

```diff
@@ -230,6 +230,18 @@ def _on_row_selected(self, event):
         selected_index = event['new']
         mos_data = self.session.data_collection['MOS Table']
 
+        # plugin data entries: select all in new row, deselect all others
+        for data_item in self.jdaviz_app.data_collection:
+            if data_item.meta.get('Plugin') is not None:
+                if data_item.meta.get('mosviz_row') == selected_index:
+                    add_data_to_viewer_message = AddDataToViewerMessage(
+                        'spectrum-viewer', data_item.label, sender=self)
+                    self.session.hub.broadcast(add_data_to_viewer_message)
```

Comment:
Save a few lines? 😸 
```suggestion
                    self.session.hub.broadcast(AddDataToViewerMessage(
                        'spectrum-viewer', data_item.label, sender=self))
```

Reply:
we don't use this pattern anywhere else, so I avoided it.... but might as well start!